pkg/driver/mount.go

```diff
@@ -39,11 +39,11 @@ const (
 	defaultRegionEnv   = "AWS_DEFAULT_REGION"
 	stsEndpointsEnv    = "AWS_STS_REGIONAL_ENDPOINTS"
 	MountS3PathEnv     = "MOUNT_S3_PATH"
+	hostTokenPath      = "HOST_TOKEN_PATH"
```

Comment:
This doesn't look right. I would expect another code change that reads the environment variable. Unless mountpoint is reading the env var for us. Either way make sure this works on a clean cluster with service account auth.

Reply:
latest commit wasn't pushed, should be there now!